doc/userguide/rules/file-keywords.rst

```diff
@@ -18,6 +18,8 @@ Example::
 
   filename:"secret";
 
+``file.name`` supports multiple buffer matching, see :doc:`multi-buffer-matching`.
```

Comment:
How did you list all these ? (Did you run a git grep ?)

Reply:
> How did you list all these ? (Did you run a git grep ?)

I looked at the commit history for the patch set, though now I realize I would have missed any later commits. I went back and looked again and did miss something. So I will have an additional commit to this PR for review.

I do have a question about:
src/detect-quic-cyu-hash.c:    DetectBufferTypeSupportsMultiInstance(BUFFER_NAME);
src/detect-quic-cyu-string.c:    DetectBufferTypeSupportsMultiInstance(BUFFER_NAME);

Should () contain the respective quic buffer names instead of BUFFER_NAME?

Reply:
There must be a macro #define BUFFER_NAME a few lines up...